read_batteryCheck.c

```diff
@@ -0,0 +1,20 @@
+#include "checker.h"
+
+void readtemperatureorsoc(float temperatureorsoc,int min_val,int max_val)
```

Comment:
is printing the only requirement for this method? even in this case, method name is not intuitive. It indicates that method reads values which is not right looking at the code.